Sources/SafeDIMacros/Macros/InstantiableMacro.swift

```diff
@@ -30,7 +30,11 @@ public struct InstantiableMacro: MemberMacro {
         providingMembersOf declaration: some DeclGroupSyntax,
         in context: some MacroExpansionContext
     ) throws -> [DeclSyntax] {
-        if let fulfillingAdditionalTypesArgument = declaration.attributes.instantiableMacro?.fulfillingAdditionalTypes {
+        if
+            let fulfillingAdditionalTypesArgument = (
+                declaration.attributes.instantiableMacro
+            )?.fulfillingAdditionalTypes
```

Comment:
Did you mean to have the `let` on a new line?

Reply:
Yeah when the line wraps like this I like to give it its own visual space